scalyr_agent/agent_main.py

```diff
@@ -690,7 +699,10 @@ def __detailed_status(self, data_directory, status_format="text"):
             fp.write(status_format)
 
         # Signal to the running process.  This should cause that process to write to the status file
-        result = self.__controller.request_agent_status()
+        if health_check:
+            result = self.__controller.request_agent_health_check()
```

Comment:
I don't think we should have a separate signal for the health check.  Rather, the health check should just use the normal `-v` signal but be selective about what it emits to stdout. 

Most likely, the "right" way to do this is to have the health check request the verbose status in JSON and then it just parses that JSON looking for the health check field.  It returns success or not based on that.